workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -344,10 +396,36 @@ def _step_dynamics(self) -> Diagnostics:
         # no diagnostics are computed by default
         return {}
 
-    def _compute_physics(self) -> Diagnostics:
-        self._log_debug(f"Physics Step (compute)")
-        self._fv3gfs.compute_physics()
-        # no diagnostics are computed by default
+    def _step_pre_radiation_physics(self) -> Diagnostics:
+        self._log_debug(f"Pre-radiation Physics Step")
+        self._fv3gfs.step_pre_radiation()
+        if self._radiation_wrapper is not None:
+            self._radiation_wrapper.rad_update(self.time, self._timestep)
+        return {
+            f"{name}_pre_radiation": self._state[name]
+            for name in self._states_to_output
+        }
+
+    def _step_radiation_physics(self) -> Diagnostics:
+        self._log_debug(f"Radiation Physics Step")
+        if self._radiation_wrapper is not None:
+            diagnostics: Diagnostics = self._radiation_wrapper.rad_compute(
+                self._state,
+                self._fv3gfs.get_tracer_metadata(),
+                self.time,
+                self._timestep,
+                self.rank,
+            )
+            for name, diag in diagnostics.items():
+                self._log_info(f"{name}: {diag}")
```

Comment:
Do you need this ad-hoc logging? Seems a bit messy for all the steps to print things to the console in potentially inconsistent ways.

Reply:
Will delete this.